prow/cluster/simple_crier_deployment.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
There is a deployment for crier in `crier_deployment.yaml`. I am wondering what's the reason to not reuse it.

Reply:
Similar to above comment about pipeline_deployment.yaml, https://github.com/kubernetes/test-infra/blob/master/prow/cluster/crier_deployment.yaml includes an optional `job-config` configmap

Reply:
As with the pipeline controller, this change would prevent crier from working properly on out instance so including it in `prow/cluster` is kind of misleading.

If you want to add a set of deployment files for a Prow instance that only uses tekton, consider grouping those in a separate directory to make it clear what the purpose is and so that users know what components they actually need to deploy (e.g. sinker and at least one of `hook`, `horologium`, or the `gerrit` adapter will be needed to actually trigger jobs.)

Reply:
Ah, I see that in the docs you have the user first follow `getting_started_deploy.md` so `hook`, `horologium`, etc will already be deployed.